models/fivetran_platform__audit_table.sql

```diff
@@ -2,10 +2,10 @@
     materialized='incremental',
     unique_key='unique_table_sync_key',
     partition_by={
-        'field': 'sync_start',
-        'data_type': 'timestamp',
-        'granularity': 'day'
+        'field': 'sync_start_day',
+        'data_type': 'date'
```

Comment:
Changed to use the sync_start_day since we have the field and will be easier for use with the lookback macro.

Reply:
[The default granularity](https://docs.getdbt.com/reference/resource-configs/bigquery-configs#using-table-partitioning-and-clustering) is `day`, and since we're now using a date column instead of timestamp, I thought it would be cleaner to remove the granularity config, but I'm not against adding it back if it would be helpful!